simplified-ui-catalog/src/main/java/org/nypl/simplified/ui/catalog/CatalogFeedViewModel.kt

```diff
@@ -413,8 +414,13 @@ class CatalogFeedViewModel(
           title = title
         )
 
-      is CatalogFeedArgumentsLocalBooks ->
-        throw IllegalStateException("Cannot transition from a local feed to a remote feed.")
+      is CatalogFeedArgumentsLocalBooks -> {
+        val oldTitle = this.feedArguments.title
+        val newTitle = title
+        throw IllegalStateException(String.format(
+          Locale.US, "Can't transition local to remote feed: {} -> {}", oldTitle, newTitle
```

Comment:
Wrong kind of format string. You want "%s -> %s".

Reply:
Ah, damn. Good catch!